rasa/core/tracker_store.py

```diff
@@ -68,38 +66,51 @@ def __init__(
         self,
         domain: Optional[Domain],
         event_broker: Optional[EventBroker] = None,
-        retrieve_events_from_previous_conversation_sessions: bool = False,
+        **kwargs: Dict[Text, Any],
     ) -> None:
         """Create a TrackerStore.
 
         Args:
             domain: The `Domain` to initialize the `DialogueStateTracker`.
             event_broker: An event broker to publish any new events to another
                 destination.
-            retrieve_events_from_previous_conversation_sessions: If `True`, `retrieve`
-                will return all events (even if they are from a previous conversation
-                session). This setting only applies to `TrackerStore`s which usually
-                would only return events for the latest session.
+            kwargs: Additional kwargs.
         """
         self.domain = domain
         self.event_broker = event_broker
         self.max_event_history = None
-        self.load_events_from_previous_conversation_sessions = (
-            retrieve_events_from_previous_conversation_sessions
+
+        # TODO: Remove this in Rasa Open Source 3.0
+        self.retrieve_events_from_previous_conversation_sessions: Optional[bool] = None
+        self._set_deprecated_kwargs_and_emit_warning(kwargs)
+
+    def _set_deprecated_kwargs_and_emit_warning(self, kwargs: Dict[Text, Any]) -> None:
+        retrieve_events_from_previous_conversation_sessions = kwargs.get(
+            "retrieve_events_from_previous_conversation_sessions"
         )
 
+        if retrieve_events_from_previous_conversation_sessions:
```

Comment:
`False` shouldn't be passed in any longer by default right? So any usage of this flag should be deprecated.
```suggestion
        if retrieve_events_from_previous_conversation_sessions is not None:
```